build_tools/get_tag.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+import os
+
+# This file assumes that our tags are always in this format: vX.X.X.
+# In that case, we would only want to write X.X.X
+
+OUT_FILE = Path(__file__).parent / 'VERSION'
+
+# Circle is easy, since they give us the git tag
+if os.getenv('CIRCLECI', False) and os.getenv('GIT_TAG', False):
+    print('Tagged commit on Circle CI. Writing to {0}'.format(OUT_FILE))
+    with open(OUT_FILE, 'w') as file:
+        file.write(os.getenv('GIT_TAG')[1:])
```

Comment:
I see a day in the future where we leave the `v` off the tag either intentionally or accidentally, and then get bitten badly. Just to be sure, let's do something like:

```python
with open(OUT_FILE, 'w') as file:
    tag = os.getenv('GIT_TAG')
    file.write(tag[1:] if tag.startswith('v') else tag)
```